Trellix/trellix-edr/tests/threat_event.json

```
@@ -0,0 +1,47 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "trellix-edr",
+        "dialect_uuid": "954a6488-6394-4385-8427-621541e881d5"
+      }
+    },
+    "message": "{\"type\":\"threats\",\"id\":\"182612\",\"attributes\":{\"aggregationKey\":\"P_6E2918727CBB836F4D8E3404BDE9AEAF5D4DED5DD1F6916AAD3F3B956E6D8A17\",\"severity\":\"s4\",\"rank\":270,\"score\":70,\"name\":\"POWERSHELL_56039776.EXE\",\"type\":\"pe\",\"status\":\"new\",\"firstDetected\":\"2023-08-27T05:34:29Z\",\"lastDetected\":\"2023-08-27T05:34:29Z\",\"hashes\":{\"sha256\":\"6E2918727CBB836F4D8E3404BDE9AEAF5D4DED5DD1F6916AAD3F3B956E6D8A17\",\"sha1\":\"D9FBB3BD6269FE3D5F349A7569964DCD1AA229B5\",\"md5\":\"6FEE39009EA5B1110C5DA6DF2B7BDC43\"}}}"
+  },
+  "expected": {
+    "message": "{\"type\":\"threats\",\"id\":\"182612\",\"attributes\":{\"aggregationKey\":\"P_6E2918727CBB836F4D8E3404BDE9AEAF5D4DED5DD1F6916AAD3F3B956E6D8A17\",\"severity\":\"s4\",\"rank\":270,\"score\":70,\"name\":\"POWERSHELL_56039776.EXE\",\"type\":\"pe\",\"status\":\"new\",\"firstDetected\":\"2023-08-27T05:34:29Z\",\"lastDetected\":\"2023-08-27T05:34:29Z\",\"hashes\":{\"sha256\":\"6E2918727CBB836F4D8E3404BDE9AEAF5D4DED5DD1F6916AAD3F3B956E6D8A17\",\"sha1\":\"D9FBB3BD6269FE3D5F349A7569964DCD1AA229B5\",\"md5\":\"6FEE39009EA5B1110C5DA6DF2B7BDC43\"}}}",
+    "event": {
+      "end": "2023-08-27T05:34:29Z",
+      "kind": "event",
+      "start": "2023-08-27T05:34:29Z"
```

I suggest adding `event.category` and `event.type`:
```suggestion
      "start": "2023-08-27T05:34:29Z",
      "category": ["intrusion_detection"],
      "type": ["info"]
```